dumb/src/it/readpsm.c

```diff
@@ -1246,7 +1246,8 @@ int pattcmp( const unsigned char * a, const unsigned char * b, size_t l )
 	}
 
 	if ( i < j ) return -1;
-	else if ( j > i ) return 1;
+	/* It's same as above. Typo? */
```

Comment:
Still, I don't get it. This statement is look like this:
```
if      ( i < j ) return -1;
else if ( i < j ) return  1;
```
Second line will never go active. Maybe there should be
```
if      ( i < j ) return -1;
else if ( i > j ) return  1;
```
?